Library/Homebrew/cmd/uses.rb

```diff
@@ -37,6 +37,8 @@ class UnavailableFormula < T::Struct
         switch "--eval-all",
                description: "Evaluate all available formulae and casks, whether installed or not, to show " \
                             "their dependents."
+        switch "--include-implicit",
+               description: "Include formulae that specify <formula> as a `:build` dependency."
```

Comment:
This seems to be wrong?

Reply:
Oops, I think I copy-pasted from somewhere else and forgot to update the description.